MapleServer2/MapleServer.cs

```diff
@@ -58,6 +60,25 @@ public static class MapleServer {
             }
         }
 
+        public static void BroadcastPacketAll(Packet packet, GameSession sender = null)
```

Comment:
Instead of using it here, calling methods on the same GameSession would be good for now, or make a new Class to handle Request of Sessions.

Reply:
Why don't use `public void` instead of `public static void` ? in this way you can add the method in the GameSession and can be called on the UserChatHandler.

Reply:
Checking again the code, FieldManager has the same methods, instead of making new one, should be use the one that FieldManager has

Reply:
The Broadcast methods in FieldManager are only for the current field, it does not include all players in the server. I'll look into making a Session controller, but not sure how to implement keeping track of all sessions outside of GameServer